cumulusci/cli/cci.py

```diff
@@ -811,6 +810,26 @@ def org_connect(config, org_name, sandbox, login_url, default, global_org):
     oauth_dict = oauth_capture()
     org_config = OrgConfig(oauth_dict, org_name)
     org_config.load_userinfo()
+    org_config._load_orginfo()
+    if (
+        org_config.organization_sobject
+        and "TrialExpirationDate" in org_config.organization_sobject.keys()
+    ):
+        if org_config.organization_sobject["TrialExpirationDate"] is None:
+            org_config.config["expires"] = "Persistent"
+        else:
+            print(
+                type(
+                    parse_api_datetime(
+                        org_config.organization_sobject["TrialExpirationDate"]
+                    )
+                )
+            )
+            org_config.config["expires"] = parse_api_datetime(
+                org_config.organization_sobject["TrialExpirationDate"]
+            ).date()
```

Comment:
The test coverage report (https://coveralls.io/builds/27073372/) shows that lines 818-830 do not have coverage. So basically that means that there aren't any tests which confirm that the correct values are stored in the org config when connecting an org that has a TrialExpirationDate.

You'll want to add 2 new tests in test_cci.py based on the existing test_org_connect test. Add a new fake response for the call to get the organization sobject:

        responses.add(
            method="GET",
            url="https://instance/services/data/v45.0/sobjects/Organization/None",
            json={"TrialExpirationDate": None},
            status=200,
        )

In the other new test, make it return a date in the Salesforce format.

As a reminder, you can check test coverage locally by running `make coverage`